akka-docs/src/main/paradox/java/howto.md

```diff
@@ -8,8 +8,117 @@ would be nice if every Akka user who finds a recurring pattern in his or her
 code could share it for the profit of all. Where applicable it might also make
 sense to add to the `akka.pattern` package for creating an [OTP-like library](http://www.erlang.org/doc/man_index.html).
 
+@@@ div { .group-java }
+
 You might find some of the patterns described in the Scala chapter of 
-@ref:[HowTo: Common Patterns](howto.md) useful even though the example code is written in Scala.
+@ref:[HowTo: Common Patterns](../scala/howto.md) useful even though the example code is written in Scala.
```

Comment:
The line is enclosed in a `@@@ div { .group-java }` block.
![image](https://user-images.githubusercontent.com/7414320/27132652-978956f4-514a-11e7-9601-8aa2a278375d.png)

The reference is intended to jump from `java/howto.md` to `scala/howto.md`. 

Instead, if we replace `(../scala/howto.md)` with `(howto.md)`, then you jump from` java/howto.md` to `java/howto.md`, the same page, which is not correct.